recipes/pynvjitlink/meta.yaml

```
@@ -0,0 +1,71 @@
+{% set name = "pynvjitlink" %}
+{% set version = "0.1.0.dev1" %}
+
+package:
+  name: pynvjitlink
+  version: {{ version }}
+
+source:
+  git_url: git@github.com:rapidsai/nvjitlink.git
```

This isn't going to work as `git` has no `ssh` keys configured

```suggestion
  git_url: https://github.com/rapidsai/nvjitlink
```